src/views/login/LoginView.tsx

```diff
@@ -61,9 +61,13 @@ export function LoginView() {
     },
   });
 
+  // Destructure formik properties
+  const { values, isValid, submitForm, setFieldValue } = formik;
+
   const handleChange = (e: React.ChangeEvent<HTMLInputElement>) => {
     const upperCaseValue = e.target.value.toUpperCase();
-    void formik.setFieldValue("code", upperCaseValue);
+    // Set shouldValidate to false to avoid returning a Promise
+    void setFieldValue("code", upperCaseValue, false);
```

Comment:
What's the meaning of `void` here?

Reply:
Ignoring the promise. No async operation there so I think there's nothig that could fail here